src/routes/paymentsCheckout.js

```diff
@@ -0,0 +1,40 @@
+const express = require("express");
+const mercadopago = require("mercadopago");
+const router = express.Router();
+require("dotenv").config();
+
+
+router.post("/payment_mercadopago", (req, res) => {
+    mercadopago.configurations.setAccessToken(process.env.ACCESS_TOKEN_MP);
+    const payment_data = {

```

Comment:
Let's split this function a usecase. There we can just pass the request body in a single const

Reply:
Hi Emi, I have a doubts about this. 

Regarding the construction of the array of products, should we obtain the data from the shopping cart, right? With that said, would a shopping cart model be created? And here, would an await function be waiting to obtain the shopping carts? 

I suppose the array would be filled when the user clicks on "pay", the data would be pulled from there, the array would be filled, and the body would be passed. 

If that's the case, would this be the correct logic?